src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+
+body,
+p {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  align-items: center;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
The border width and border-radius are hardcoded values. Consider using variables for these values to maintain consistency and ease of maintenance, especially if these styles are repeated or need to be consistent across components. For example, `$border-width: 1px;` and `$border-radius: 5px;`.